src/components/DynamicForm.tsx

```diff
@@ -67,55 +72,76 @@ const DynamicForm: React.FC<DynamicFormProps> = ({
     onError(errors);
   };
 
-  function transformErrors(errors: any) {
+  const handleSubmit = (
+    event: IChangeEvent<any, RJSFSchema, any>,
+    formEvent: React.FormEvent<any>
+  ) => {
+    console.log("Submit button clicked");
+    setSubmitted(true);
+    onSubmit(event, formEvent);
+  };
+
+  const handleChange = (event: IChangeEvent<any>) => {
+    const cleanAndReplace = (data: any) => {
+      for (const key in data) {
+        if (Array.isArray(data[key])) {
+          data[key] = Array.from(
+            new Set(
+              data[key]
+                .filter((item: any) => item !== "")
+                .map((item: any) => (item === "_lifeskills" ? "life_skills" : item))
```

Comment:
Don't know sir, for this life_skills field only some user have set   _lifeskills so while updating field it get wrong value, thats why set hardcoded value here.

Reply:
It should not be hardcoded, check for other solutions

Reply:
Yes sir, now I have removed this hardcoded value check create and update flow for facilitator working as accepted...faced that problem because of sanket had made changes db... but now working as accepted